src/index.js

```diff
@@ -48,6 +48,17 @@ function testUniqueness( element, selector )
 }
 
 /**
+ * Tests all selectors for uniqueness and returns the first unique selector.
+ * @param  { Object } element
+ * @param  { Array } selectors
+ * @return { String }
+ */
+function getFirstUnique( element, selectors )
+{
+    return selectors.find( testUniqueness.bind( null, element ) );
```

Comment:
According to https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/find the callback receives element as the first argument.